### DISCUSSION TITLE
New tool for previewing fzf options/colors

### DISCUSSION THREAD

**Discussion by vitormv** [junegunn/fzf#3592] [Show and tell]
Hi folks! First of all, thanks for creating this awesome tool! I liked this project so much, that I wanted to give something back to the community, but as a mostly Frontend developer, go is way out of my comfort zone, therefore I decided to do something more with the skills I already have (and also a fun side-project to try out the Svelte framework).

We already had a [fzf color picker](https://minsw.github.io/fzf-color-picker/), although it was missing a lot of colors from fzf. So I decided to create a new one that covers the whole Color Options Palette of `fzf`, but also add a quick way to preview some of the options as well, in a intuitive interface heavily inspired by [terminal.sexy](https://terminal.sexy/).

![](https://github.com/vitormv/fzf-themes/raw/main/screenshot.png)

This is still pretty much fresh out of the oven, so there are surely bugs. But wanted to post it here to maybe get some additional feedback/ideas/suggestions etc.

Also the idea was to create a tool to easily tweak/preview things, and NOT to have a 1:1 replica of fzf. So some fzf options are surely missing, but I guess this is a good starting point.

**DEMO:** https://vitormv.github.io/fzf-themes/
**REPO:** https://github.com/vitormv/fzf-themes

Feedback welcome!

**Comment by junegunn** [comment]
Wow, this is great, thanks for creating it! Do you mind if I add this to README.md or ADVANCED.md?

**Reply by vitormv** [reply]
no, thank YOU for making `fzf` in the first place :) And of course you can link to the project anywhere :)
